Formula/couchdb.rb

```diff
@@ -19,14 +19,32 @@ class Couchdb < Formula
   depends_on "erlang@22" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
+  depends_on "gcc"
   depends_on "icu4c"
   depends_on "openssl@1.1"
   depends_on "spidermonkey"
 
   conflicts_with "ejabberd", because: "both install `jiffy` lib"
 
+  fails_with :clang # c++20 support
+
+  patch :DATA
```

Comment:
Can you include a comment explaining the source of the patch and when it can be removed?